docs/tsconfig.md

```diff
@@ -23,6 +24,25 @@ i.e. an empty JSON file at the *root* of your project :heart: This will be suffi
 
 ## Examples
 
+### exclude
+These references are relative to the `tsconfig.json` path. This does not accept glob formatting.
+
+NOTE: `files` can override the `exclude` array. It's recommended to use either `exclude` & `files` -- OR -- `filesGlob`, not both. `filesGlob` solves the exclude problem as long as your team is exclusively using atom-typescript.
```

Comment:
I totally space the mutual exclusivity of `files` and `exclude`. Thanks for the gentle reminder. I'll make change.